### PR TITLE
Adds Header with Search Bar

### PR DESCRIPTION
**Pull request by astr0-4**
* Connects search bar input text to redux
 * Connecting submit button to redux

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Good work! Looks great!

File: src/redux/reducers/header.reducer.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export default ( state = '', action) => {
+	switch (action.type) {
+		case HEADER_ACTIONS.UPDATE_SEARCH_TEXT:
+			return 	{...state, searchText: action.payload}
+		case HEADER_ACTIONS.GET_ALCOHOL_LIST:
+			return {...state, alcohol: action.payload}
+		default:
+		 return state
+	}
+}
```
Comment (jprittie) [inline src/redux/reducers/header.reducer.js]: Hi... the search text is updating on state just fine. I would say that we don't also need to set 'alcohol' on state, i.e., we don't need this part of the reducer:
`case HEADER_ACTIONS.GET_ALCOHOL_LIST:
       return {...state, alcohol: action.payload}`

I think that the action GET_ALCOHOL_LIST is what the epic will listen for. Then the epic will make the API call and return a response with one of two actions - one action for success, one action for failure. Those epics will then go to a reducer that will set search results on state. Does that make sense?

Comment (astr0-4) [reply src/redux/reducers/header.reducer.js]: yes that makes sense @jprittie! I'll remove that

**Review by jprittie — APPROVED** [review]
Good to merge, I think!

**Pull request merged**